.github/workflows/sync-warp.yaml

```
@@ -0,0 +1,68 @@
+#
+# dotfiles
+# Sync WARP VM's dotfiles
+#
+
+name: Sync dotfiles on WARP
+on:
+  push: {}
+    #branches: [main]
```

```suggestion
    branches: [main]
```
Revert after testing.